css/app.css

```diff
@@ -215,6 +215,18 @@ html, body {
   padding-left: 3px;
 }
 
+#graphiql-container .tool-button {
+  cursor: pointer;
+  text-decoration: none;
+  font-family: sans-serif;
+  color: #ccc;
+  padding: 2px 4px;
+  margin: 0 4px;
+}
+#graphiql-container .tool-button:hover {
+  color: #333;
```

Comment:
I think :hover need to stay as it's a feedback that the button is clickable. that said, I can make the button changing color when there is an error?